go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/containers/gvisor-tap-vsock
 
-go 1.22.0
+go 1.23.7
```

Comment:
Can we stick to 1.23.0? I already don’t like that our dependencies force major go updates on us, I’d prefer to stick to the minimum version possible, and let people building gvisor-tap-vsock use the go version they prefer (ideally the latest released)

Reply:
Done